[2024-09-25T09:45:17.504+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T09:45:17.527+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branch_python_operator_example.branching manual__2024-09-25T09:45:15.234580+00:00 [queued]>
[2024-09-25T09:45:17.539+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branch_python_operator_example.branching manual__2024-09-25T09:45:15.234580+00:00 [queued]>
[2024-09-25T09:45:17.540+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-25T09:45:17.559+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): branching> on 2024-09-25 09:45:15.234580+00:00
[2024-09-25T09:45:17.566+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=117) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T09:45:17.569+0000] {standard_task_runner.py:72} INFO - Started process 118 to run task
[2024-09-25T09:45:17.570+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'branch_python_operator_example', 'branching', 'manual__2024-09-25T09:45:15.234580+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/materi/branch_operator.py', '--cfg-path', '/tmp/tmpquoxy9ze']
[2024-09-25T09:45:17.571+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask branching
[2024-09-25T09:45:17.625+0000] {task_command.py:467} INFO - Running <TaskInstance: branch_python_operator_example.branching manual__2024-09-25T09:45:15.234580+00:00 [running]> on host 639d778bfb8f
[2024-09-25T09:45:17.726+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='branch_python_operator_example' AIRFLOW_CTX_TASK_ID='branching' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T09:45:15.234580+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T09:45:15.234580+00:00'
[2024-09-25T09:45:17.727+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T09:45:17.746+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-09-25T09:45:17.747+0000] {python.py:240} INFO - Done. Returned value was: odd_branch
[2024-09-25T09:45:17.747+0000] {branch.py:38} INFO - Branch into odd_branch
[2024-09-25T09:45:17.748+0000] {skipmixin.py:230} INFO - Following branch ('odd_branch',)
[2024-09-25T09:45:17.760+0000] {skipmixin.py:278} INFO - Skipping tasks [('even_branch', -1)]
[2024-09-25T09:45:17.800+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T09:45:17.801+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=branch_python_operator_example, task_id=branching, run_id=manual__2024-09-25T09:45:15.234580+00:00, execution_date=20240925T094515, start_date=20240925T094517, end_date=20240925T094517
[2024-09-25T09:45:17.825+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T09:45:17.864+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T09:45:17.867+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
